### DISCUSSION TITLE
How to get all the followig from my account?

### DISCUSSION THREAD

**Discussion by nalindev** [MarshalX/atproto#496] [Questions] [answered]
I am trying to get the following from my account but this code is not working for me and throwing an error. can someone please help me with this issue?

My code -
```
def main() -> None:
    client = Client()
    client.login('nalinn.bsky.social', 'Nalin@10027')
    #print(client.app.bsky.graph.follow.get(client.me.did))

    print('Home (Following):\n')
    handle = 'nalinn.bsky.social'
    followers = client.app.bsky.graph.get_follows(actor=handle)
    print(followers)
```
Error -
![image](https://github.com/user-attachments/assets/8bbfd3cc-c136-4fac-b602-476557a32db4)

**Comment by basinbasin** [comment]
Does handle_cursor_pagination.py in https://github.com/MarshalX/atproto/tree/main/examples/advanced_usage give you what you need?

**Reply by nalindev** [reply]
Hello @basinbasin No, I’m not trying to paginate all the data here. My goal is to first retrieve the following list(eg., it can be 5/10/20/x), but I’m not getting any response. I’m unsure about the arguments that need to be passed inside ```client.app.bsky.graph.get_follows()``` method. Could you please provide an example to help me?

![image](https://github.com/user-attachments/assets/1f39e65d-a362-48e3-860c-3ba433bd3c86)

**Comment by basinbasin** [comment, marked as answer]
That page has an example where a cursor is used as the second parameter in the get_follows call:
```

def main() -> None:
    # This is an example for get_follows method.
    client = Client()
    username = ""
    password = ""
    client.login(username, password)
    handle = username

    cursor = None
    follows = []

    while True:
        fetched = client.get_follows(actor=handle, cursor=cursor)
        follows = follows + fetched.follows

        if not fetched.cursor:
            break

        cursor = fetched.cursor

    print(follows)
```
Does that help at all?

**Reply by nalindev** [reply]
yes it is working
